src/main/java/com/conveyal/r5/analyst/ImprovementProbabilityGridSampler.java

```diff
@@ -0,0 +1,108 @@
+package com.conveyal.r5.analyst;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.S3Object;
+import com.google.common.io.LittleEndianDataInputStream;
+
+import java.io.IOException;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Sample from an access grid of instantaneous accessibility to compute the probability of improvement.
```

Comment:
Hmm, the grid of distributions is an interesting thought. Then you could say "what's the probability of an improvement greater than X"